specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/managedClusters.json

```diff
@@ -3556,9 +3556,90 @@
         "clientId"
       ]
     },
+    "FlexProfile": {
+      "type": "object",
+      "properties": {
+        "scaleProfile": {
```

Comment:
very minor: For whatever reason we often try to avoid nested profiles.

I don't really have a problem with it, but if you prefer it we could call this just `scale` instead (that's all I mean by nested profiles, we sometimes avoid the _word_ profile nested in a thing that's already a profile.

Reply:
I updated the field name to be scale but keep the ref type as scaleProfile to make it clearer.